peachjam/js/components/FindDocuments/SearchResult.vue

```diff
@@ -15,7 +15,11 @@
       <i>{{ item.citation }}</i>
     </div>
     <div class="text-muted">
-      {{ item.date }}
+      {{ item.date }} &nbsp; {{ item.doc_type }}
```

Comment:
Please can you put the doc type in a `<span class="ms-3">` rather than using the nbsp, so that there is consistent spacing between the items.

Reply:
I'm also not sure why we're using `strong` below 🤷  can you change both `strong`s to `span`s.

Reply:
less distracting formatting :)